spec/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../book/book'
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT** **In the README.md file:** I think it would be better if you update your README file by adding a **## Test** section with some hints on how to install the `respec`  gem and run `Tests`. This will help users without information about `respec.` I know you can do that :muscle: :muscle: